airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -184,15 +184,17 @@ def get_conn(self) -> BlobServiceClient:
             # connection_string auth takes priority
             return BlobServiceClient.from_connection_string(connection_string, **extra)
 
+        account_url = conn.host if conn.host else f"https://{conn.login}.blob.core.windows.net/"
+        if not account_url.startswith("https://"):
+            account_url = f"https://{conn.login}.blob.core.windows.net/"
```

Comment:
Seems like this could be simplified a few different ways. 

Consider one of the following:

1. Making the `else` on L187 either an empty string or None, which would trigger the `if` block below it.
2. Using a variable to hold the repeated "magic string".
3. `account_url = conn.host if conn.host.startswith("https://") else f"https://{conn.login}.blob.core.windows.net/"`